retention/src/test/java/org/apache/falcon/retention/FeedEvictorTest.java

```diff
@@ -462,6 +462,43 @@ public void testFeedBasePathExists() throws Exception {
         }
     }
 
+    @Test
+    public void testEvictionStatsMetaWithNoPattern() throws Exception {
+        try {
+            Configuration conf = cluster.getConf();
+            FileSystem fs = FileSystem.get(conf);
+            fs.delete(new Path("/"), true);
```

Comment:
@bvellanki there are other tests doing this and it seems to work. If there is any race condition, all the tests in this class need to be sequenced one after the other. I'm not planning to address it now.

   Configuration conf = cluster.getConf();
+            FileSystem fs = FileSystem.get(conf);
+            fs.delete(new Path("/"), true);